Doc/library/dataclasses.rst

```diff
@@ -84,12 +84,12 @@ Module contents
      class C:
          ...
 
-   The parameters to :func:`dataclass` are:
+   The parameters to :func:`!@dataclass` are:
 
    - ``init``: If true (the default), a :meth:`~object.__init__` method will be
      generated.
 
-     If the class already defines :meth:`~object.__init__`, this parameter is
+     If the class already defines :meth:`~!object.__init__`, this parameter is
```

Comment:
renders the same, but slightly simpler for readers of the unrendered RST file:

```suggestion
     If the class already defines :meth:`!__init__`, this parameter is
```

Reply:
Hmm, yes, simpler to read in the .rst, but not semantically accurate, since this is a reference to the (non-existent) `dataclasses.__init__` method?

Reply:
From Sphinx's point of view, I suppose so , but does that really matter? It will render the same in the documentation displayed to readers.

Reply:
There doesn't seem to be a way to make the longer form pass CI, so I'll take the shorter-form changes.